client/lib/posts/post-list-store.js

```diff
@@ -1,23 +1,28 @@
 /**
  * External dependencies
  */
-var debug = require( 'debug' )( 'calypso:posts-list' ),
-	clone = require( 'lodash/lang/clone' ),
-	assign = require( 'lodash/object/assign' ),
-	transform = require( 'lodash/object/transform' ),
-	difference = require( 'lodash/array/difference' ),
-	last = require( 'lodash/array/last' ),
-	max = require( 'lodash/collection/max' );
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:posts-list' );
```

Comment:
Nitpicking now, but I'd still keep the dependencies blocks for `import`s only and move this assignment to `Module variables`, though I'm not sure whether there's any precedent here.